locallib.php

```diff
@@ -485,7 +485,7 @@ function local_bongo_unregister_bongo_integration() {
     $bongoconfig = get_config('local_bongo');
     $siteconfig = get_config('');
 
-    if ($bongoconfig == NULL || is_null($bongoconfig) || is_null($bongoconfig->name)) {
```

Comment:
Rather than is_null, check if the value is set or not.  This was the reinstall defect.